### ISSUE TITLE
Add rcloneExplorer to F-Droid

### ISSUE THREAD

**Issue by FoxP** [kaczmarkiewiczp/rcloneExplorer#197]
**Just and idea :** Add rcloneExplorer to [F-Droid](https://f-droid.org/en/docs/FAQ_-_App_Developers/).

**Comment by kaczmarkiewiczp** [comment]
I had a quick look at F-Droid FAQ and how to include an app. If I'm understanding it correctly, the process doesn't require much input from me (which is nice as currently I don't have as much time to work on this project as I used to).

Looking at the [inclusion policy](https://f-droid.org/docs/Inclusion_Policy/), I came across two issues about which I'm not sure. 
F-droid doesn't allow proprietary analytic dependencies (like Firebase), which rclone explorer uses. Would I be required to maintain a separate branch or fork without Firebase?

One of the points also states that "Binary dependencies such as JAR files have to be replaced by source-built versions or used from a trusted repository". I'm not sure how the rclone binaries present in the project would tie in with that point. I manually compile rclone binaries using Termux on three different devices (ARM32, ARM64, and x86) since rclone doesn't provide binaries that are compatible with Android.

Also, when I generate the APK, I create four different releases; one compatible with ARM32 devices, one compatible with ARM64 devices, one compatible with x86 devices, and one compatible with all three types of devices. This requires me to delete the unnecessary rclone binaries from the project folder, and generate signed APK for each device type.

I'm just not sure how all of that would work on F-Droid. If anyone familiar with F-Droid could answer those questions, I would greatly appreciate the help.

**Comment by FoxP** [comment]
You are right, you should maintain a separate branch without proprietary code / resources. 

For the rest, maybe you should ask the developers: https://f-droid.org/en/docs/FAQ_-_General/#where-should-i-ask-questions

**Comment by Poussinou** [comment]
Tagging @Rudloff for more information

**Comment by Rudloff** [comment]
Instead of a separate branch, we usually recommend using [build variants](https://developer.android.com/studio/build/build-variants), which is often easier to maintain.

As for the rclone binaries, it should not be a problem as long as we can rebuild these from source ourselves with the NDK. But I think it would require some work on our end to automate the build.

**Comment by joserebelo** [comment]
Just for reference, [Syncthing](https://github.com/syncthing/syncthing-android) should have similar build requirements. It uses the syncthing backend in Go, which is built from source using the NDK, and it is currently on F-Droid, so I would think the build process should be quite similar and adaptable for rcloneExplorer.

**Comment by MarkusLitz** [comment]
Are these binaries NaCL binaries of the official rclone code?

**Comment by phly95** [comment]
Turns out, using Aurora store, it thinks this app is an update to rclone explorer. Definitely a sketchy looking app.

https://play.google.com/store/apps/details?id=ca.pkay.rcloneexplorer

**Comment by FoxP** [comment]
Same on Yalp Store. No source code available for this "fork"?

**Comment by beddari** [comment]
For reference, I don't know anything about this, but ... 
https://play.google.com/store/apps/details?id=pl.sviete.dom.rcloneexplorer
https://github.com/sviete/AIS-synchro
